Source/CLI/Help.cpp

```diff
@@ -11,18 +11,53 @@
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
+#include "Common/Core.h"
 #include "Help.h"
 #include "Config.h"
 //---------------------------------------------------------------------------
 
 //***************************************************************************
 //
 //***************************************************************************
+//---------------------------------------------------------------------------
+String Program_Name(const String &Set)
+{
+    static String Program;
+
+    if(!Set.empty())
+    {
+        //Set Program Name
+        Program = Set;
+        #if defined(_MSC_VER)
+            Program = Program.substr(Program.rfind('\\')+1);
+            Program = Program.substr(0, Program.find('.'));
+        #else
+            Program = Program.substr(Program.rfind('/')+1);
+        #endif
+    }
+
+    if(Program.empty())
+    {
+        #if defined(UNICODE)

```

Comment:
for all "UNICODE" stuff:
there is a "__T()" define, especially for not repeating code (is "L" with Unicode, "" with non Unicode)